src/python/pants/backend/jvm/tasks/jvm_compile/zinc/zinc_compile.py

```diff
@@ -322,6 +322,11 @@ def relative_to_exec_root(path):
     zinc_args.append('-transactional')
 
     compiler_option_sets_args = self.get_merged_args_for_compiler_option_sets(compiler_option_sets)
+
+    # Needed to make scoverage CodeGrid highlighting work
```

Comment:
This should be incorporated in `ScalaLibrary` rather than here.

Reply:
How can I set `compiler_option_set_enabled_args` in ScalaLibrary. I looked at the `compiler_option_sets_mixin.py` here : https://github.com/pantsbuild/pants/blob/14e0bd6b27910310e225ff9586c2410459153fd9/src/python/pants/option/compiler_option_sets_mixin.py But there is no API for adding extra arguments. That is I added it manually here.

Reply:
Ok, fine with this as is I guess.